test/support/test_config.h

```diff
@@ -126,6 +126,9 @@
 
 #define _PSTL_MSVC_LESS_THAN_CPP20_COMPLEX_CONSTEXPR_BROKEN (_MSC_VER && __cplusplus < 202002L && _MSVC_LANG < 202002L)
 
+// In MSVC C++20, _Can_deref concept returns wrong result for sycl_iterator, resulting in errors moving sycl_iterator
+#define _PSTL_MSVC_GREATER_EQUAL_CPP20_CAN_DEREF_CONCEPT_BROKEN (_MSC_VER && __cplusplus >= 202002L && _MSVC_LANG >= 202002L)

```

Comment:
Should we check both `__cplusplus`  and `_MSVC_LANG` states here?

Reply:
I'm following the example from the macro a few lines above it.  However, I suppose either just checking `_MSVC_LANG` only or using an `||` might be better here as I look a bit further (at test_macros.h for instance).  which do you suggest?

Reply:
May be
`#define _PSTL_MSVC_GREATER_EQUAL_CPP20_CAN_DEREF_CONCEPT_BROKEN (_MSVC_LANG >= 202002L)`
is enough?